itests/karate-config.js

```diff
@@ -22,7 +22,8 @@ function fn() {
 
         !('JBANG_REPO' in env) && (env.JBANG_REPO = sc + sep + "karate-m2")
         !('JBANG_DIR' in env) && (env.JBANG_DIR = sc + sep + "karate-jbang")
-        
+        !('JBANG_NO_VERSION_CHECK' in env) && (env.JBANG_NO_VERSION_CHECK = "true")
```

Comment:
since it is named JBANG_NO_VERSION_CHECK I think we should just check for presence so doing `export JBANG_NO_VERSION_CHECK` would be enough ?

Reply:
Perhaps, I never trust my intuition on what `export JBANG_NO_VERSION_CHECK` will do :-)
The check itself in Jbang is pretty broad, anything that is not "false" (ignoring case) is considered to be "true".

Reply:
(which actually means that `export JBANG_NO_VERSION_CHECK` should just work fine)